metadb/README.md

```diff
@@ -0,0 +1,135 @@
+# metadb
+
+`metadb` is Metadata Layer, the one that store these following data in RQLite cluster database:
+- User specified data (userdata) --> Ready
+- Supernode specified data. --> Under Development
+- It also have responsible to calculate walletnode and supernode reputation scores, and take action for the low reputation user/walletnode/supernode.--> Under Development
+
+## Quick Start
+
+ 
+1. Build supernode, walletnode, tools/pastel-api with command:
+
+>cd "target  dir"
+
+>go build ./
+
+
+2. The application running path depends on the OS:
+
+* MacOS `~/Library/Application Support/Pastel`
+
+* Linux `~/.pastel`
+
+* Windows (>= Vista) `C:\Users\Username\AppData\Roaming\Pastel`
+
+* Windows (< Vista) `C:\Documents and Settings\Username\Application Data\Pastel`
+
+  
+3. Testing in local
+
+- Start 3 Supernodes:
+
+>SUPERNODE_DEBUG=1 LOAD_TFMODELS=0 ./supernode --log-level debug -c ./examples/configs/localnet-4444.yml
```

Comment:
I don't think we need LOAD_TFMODELS=0 any more.

Reply:
removed
thank you